fluss-server/src/main/java/com/alibaba/fluss/server/zk/ZooKeeperClient.java

```diff
@@ -259,12 +260,29 @@ public void deleteLeaderAndIsr(TableBucket tableBucket) throws Exception {
     // --------------------------------------------------------------------------------------------
 
     /** Register a database to zk. */
+    @Deprecated
```

Comment:
ZookeeperClient is an internal class, we don't need to keep compatibility. Just remove the legacy methods.